Jenkinsfile

```diff
@@ -1231,60 +1096,46 @@ pipeline {
                         beforeAgent true
                         allOf {
                             not { branch 'weekly-testing' }
-                            not { environment name: 'CHANGE_TARGET', value: 'weekly-testing' }
+                            not { environment name: 'CHANGE_TARGET',
+                                              value: 'weekly-testing' }
                             expression { ! skip_stage('test-centos-rpms') }
                         }
                     }
                     agent {
                         label 'ci_vm1'
                     }
                     steps {
-                        provisionNodes NODELIST: env.NODELIST,
-                                       node_count: 1,
-                                       profile: 'daos_ci',
-                                       distro: 'el7',
-                                       snapshot: true,
-                                       inst_repos: el7_daos_repos(),
-                                       inst_rpms: 'environment-modules'
-                        runTest script: "${rpm_test_pre}" +
-                                        "sudo yum -y install daos{,-client}-${daos_packages_version("centos7")}\n" +
-                                        "sudo yum -y history rollback last-1\n" +
-                                        "sudo yum -y install daos{,-{server,client}}-${daos_packages_version('centos7')}\n" +
-                                        "sudo yum -y install daos{,-tests}-${daos_packages_version("centos7")}\n" +
-                                        "${rpm_test_daos_test}" + '"',
-                                junit_files: null,
-                                failure_artifacts: env.STAGE_NAME, ignore_failure: true
-                    }
+                        testRpm node_count: 1,
+                                profile: 'daos_ci',
+                                distro: 'centos7',
+                                inst_repos: el7_daos_repos(),
+                                daos_pkg_version: daos_packages_version("centos7"),
+                                ignore_failure: true
```

Comment:
Why are we ignoring a failure of this stage?

Reply:
No idea.  You can see it there in the existing code above it.

It stands out more in the refactor.

I think we can also default node_count, profile, and distro.

Reply:
> No idea. You can see it there in the existing code above it.

Not quite following.  What existing code is allowing this stage to fail?  Regardless, it shouldn't be allowed to fail.

> I think we can also default node_count, profile, and distro.

That sounds like a good idea.  Why not do it in this PR?